release/pipelines/windows-efi-installer/windows-efi-installer.yaml

```diff
@@ -290,6 +327,25 @@ spec:
             value: modify-data-object
           - name: version
             value: v0.20.0
+    - name: delete-imported-configmaps
+      params:
+        - name: SCRIPT
+          value: oc delete -f $(params.autounattendXMLConfigMapsURL)
```

Comment:
Can we cache the config map somewhere? If it changes during the pipeline run, then the delete might not work as expected.

Reply:
we can use the `customizeConfigMapName` or `autounattendConfigMapName` params to delete the correct configmaps instead of URL

Reply:
Problem will be, if the URL contains multiple configmaps, then it will delete only single CM

Reply:
Can you return the created config maps as a result in `import-unattend-configmaps` and use that result to delete them in `delete-imported-configmaps`?

Reply:
I don't think so, because the task does not have results defined in manifest https://github.com/tektoncd/catalog/blob/main/task/openshift-client/0.2/openshift-client.yaml#L14

Reply:
Can you perhaphs cache the ConfigMaps file in a [Workspace](https://tekton.dev/docs/pipelines/pipelines/#specifying-workspaces) instead?

Reply:
If I am not mistaken, the workspaces requires record in PipelinerRun (see the *note* in https://tekton.dev/docs/pipelines/workspaces/#specifying-workspaces-in-pipelineruns). That means, we would break triggering of the pipeline from the UI. Users would have to build their own manifest for triggering the pipeline, which would go against the idea of this pipeline.

Reply:
Unfortunate. I'm just concerned this will be handed back to us as a possible bug in the future. If there's not other way, maybe we can at least document it in the description or somewhere?

Reply:
fair point, let me update it tomorrow

Reply:
documented in https://github.com/kubevirt/kubevirt-tekton-tasks/pull/375/commits/4bda75da33dbd232cc993260e05a7a787b692858